index.html

```diff
@@ -0,0 +1,192 @@
+            <li>
+              <a href="about.html">About</a>
+            </li>
+            <li>
+              <a href="#">Program</a>
+            </li>
+            <li>
+              <a href="#">Join</a>
+            </li>
+            <li>
+              <a href="#">Sponsor</a>
+            </li>
+            <li>
+              <a href="#">News</a>
+            </li>   
+          </ul>
+
+          <button type="button" class="nav-btn">
+            CC Campaign
+          </button>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home">
+      <header id="titles-home">
+        <img src="images/title.png" alt="Creative-Commons-Global-Summit-2015">
+        <!-- <h2>"Hello! Sharing world"</h2>
+        <h1>
+          CREATIVE COMMONS
+          <br/>
+          GLOBAL SUMMIT
+          <br/>
+          2015
+        </h1> -->
+      </header>
+
+      <div id="card-home">
+        <p>A joyful celebration believing in the value of opennes and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+        <h5>2015.10.15(THU) ~ 16(FRI)</h5>
+        <p>@ National Musuem of Koerea, Art Center Nabi and more</p>
+      </div>
+    </section>
+
+    <section class="program">
+      <header>
+        <h3>Main Program</h3>
+        <div class="vector"></div>  
+      </header>
+      
+      <ul class="card">
+        <li>
+          <img src="images/lecture.png" alt="#">
+          <a href="#">Lecture</a>
+          <p>Speakers from various countries about the messages of sharing and opening.</p>
+        </li>
+        <li>
+          <img src="images/exhibition.png" alt="#">
+          <a href="#">CC Exhibition</a>
+          <p>Appreciate various creations applying CC license of artists, organized from Art Center Nabi.</p>
+        </li>
+        <li>
+          <img src="images/forum.png" alt="#">
+          <a href="#">Forum</a>
+          <p>Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </li>
+        <li>
+          <img src="images/workshop.png" alt="#">
+          <a href="#">Workshop</a>
+          <p>Try creating you own work using open source license rather than just watching.</p>
+        </li>
+        <li>
+          <img src="images/ignite.png" alt="#">
+          <a href="#">CC Ignite</a>
+          <p>Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+        </li>
+      </ul>
+      
+      <button type="button" class=" mobile-only">Join CC Global Summit 2015</button>
+      <a id="a" class="desktop-only" href="#">SEE THE WHOLE PROGRAM</a>
+    </section>
+
+    <section class="speakers">
+      <h3>Featured Speakers</h3>
+      <div class="vector"></div>  
+
+      <div id="featured"></div>
+
+      <button type="button" class="more">
+        MORE
+        <img src="images/arrow.png" alt="more-speakers">
+      </button>
+    </section>
```

Comment:
- [ ] It would be better if in desktop size there are 2 columns instead of 1 column. I can see that there is 2 column on a big screen. kindly fix it so the tablet size and desktop size show properly. 

![Capture](https://user-images.githubusercontent.com/64173189/142532548-5dd1814b-06ec-4be1-8cd4-79fc5b5ff25f.JPG)